packages/settings/setters/toggles/lightDarkModeToggle.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import settingConstants from '@settings/configuration/settingConstants';
+import Toggle from './toggle';
+
+function LightDarkToggle({ t }) {
```

Comment:
the translation shouldn't pass from the parent, use the translation hook
```
const { t } = useTranslation();
```
https://react.i18next.com/latest/usetranslation-hook